test/functional/driver/derived-data-path-e2e-specs.js

```diff
@@ -0,0 +1,58 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { retryInterval } from 'asyncbox';
+import { getSimulator } from 'appium-ios-simulator';
+import { shutdownSimulator } from '../helpers/simulator';
+import { createDevice, deleteDevice } from 'node-simctl';
+import { MOCHA_TIMEOUT, initSession, deleteSession } from '../helpers/session';
+import { UICATALOG_SIM_CAPS } from '../desired';
+
+
+const SIM_DEVICE_NAME = 'xcuitestDriverTest';
+
+chai.use(chaiAsPromised);
+
+const deleteDeviceWithRetry = async function (udid) {
+  try {
+    await retryInterval(10, 1000, deleteDevice, udid);
+  } catch (ign) {}
+};
+
+describe('XCUITestDriver', function () {
+  this.timeout(MOCHA_TIMEOUT);
+
+  let baseCaps;
+  let caps;
+
+  let driver;
+  before(async function () {
+    const udid = await createDevice(SIM_DEVICE_NAME,
+      UICATALOG_SIM_CAPS.deviceName, UICATALOG_SIM_CAPS.platformVersion);
+    baseCaps = Object.assign({}, UICATALOG_SIM_CAPS, {udid});
+    caps = Object.assign({
+      usePrebuiltWDA: true,
+      agentPath: "/tmp/WebDriverAgent/WebDriverAgent.xcodeproj",
+      derivedDataPath: "/tmp/WebDriverAgent/DerivedData/WebDriverAgent"
+    }, baseCaps);
+  });
+  after(async function () {
+    const sim = await getSimulator(caps.udid);
+    await shutdownSimulator(sim);
+    await deleteDeviceWithRetry(caps.udid);
+  });
+
+  afterEach(async function () {
+    // try to get rid of the driver, so if a test fails the rest of the
+    // tests aren't compromised
+    await deleteSession();
+  });
+
+  if (!process.env.REAL_DEVICE) {
+    it.skip('should start and stop a session', async function () {
```

Comment:
Why skip?

Reply:
This needs to copy(or move) the WebdriverAgent folder to `/tmp`/ I use it for testing locally. Want me to copy the existing folder to `/tmp` as a before taks and dont skip it ?

Reply:
If it's not difficult to set up, I would do it, yes. :)

Reply:
Updated! Im assuming the WedDriverAgent already created the .app on previous tests.